clarity/contracts/arkadiko-stake-pool-diko-v1-1.clar

```diff
@@ -61,10 +61,15 @@
     (total-diko-supply (+ diko-supply rewards-to-add))
 
     ;; Total stDIKO supply
-    (stdiko-supply (unwrap-panic (contract-call? .stdiko-token get-total-supply)))
+    (calculated-stdiko-supply
```

Comment:
PP: Remove this logic where the parameter can be zero. On line 138, just pass the total supply instead of 0.